smartdashboard/utils/ManifestReader.py

```diff
@@ -142,6 +148,8 @@ def get_manifest(self) -> Manifest:
                 "Ensembles are malformed.", file=self._file_path, exception=exc
             ) from exc
 
+        runs = [Run(**run_data) for run_data in runs_data]
```

Comment:
I totally see what you're saying. I'm running into a problem because I need to attach `run_id` to each of the entities. If I didn't have to do that, then I'd be able to accomplish what you're recommending for sure. Maybe you already know of a way to get around that?